CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,15 @@
+# Code of Conduct
+
+PyKitOps is a sub-project of the KitOps open source project and is governed by the same [KitOps Code of Conduct](https://github.com/jozu-ai/kitops/blob/main/CODE-OF-CONDUCT.md).
```

Comment:
Calling it a "sub-project" might be a bit confusing, as it isn't a "git subproject".  kitops and pykitops are two independent git repositories, currently.